Playbooks/playbook-PAN-OS_EDL_Setup.yml

```diff
@@ -684,19 +698,20 @@ inputs:
   description: The IP address of the web server on which the files are stored. The
     web server IP address is configured in the integration instance.
 - key: pre-post-rulebase
-  value:
-    simple: pre-rulebase
+  value: {}
   required: true
   description: Either pre-rulebase or post-rulebase,  according to the rule structure.
 - key: rule-position
   value: {}
-  required: true
+  required: false
   description: |-
     The position of the rule in the ruleset. Valid values are:
       * Top
       * Bottom
       * Before
       * After
+
+    The default position will be 'Top'
```

Comment:
@yaron-libman

Reply:
@Arsenikr I think the sentence should be 'The default position is 'Top'.